images/dockerregistry/config.yml

```diff
@@ -35,3 +35,7 @@ middleware:
         blobrepositorycachettl: 10m
   storage:
     - name: openshift
+openshift:
```

Comment:
Resolved eye to eye. @legionus convinced me that this is a preferred place for our current and future configuration. As long as it's versioned properly. In the future, we'd like to move `middleware` section here completely. The `openshift` section can co-exist with the `middleware` entries for now since they have separate areas of interest. The `middleware` section gets out of hand already so I'd prefer to do the configuration refactoring sooner rather than later. I'll open an issue to track this.